app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -65,6 +65,11 @@ public function store(ReportbackRequest $request)
      */
     public function updateReportbackItems(Request $request)
     {
+        $this->validate($request, [
+            '*.rogue_reportback_item_id' => 'required',
```

Comment:
I tried to put it in it's own Request class but couldn't get it to work, even with the `*` notation. After reading a few blog posts/stack overflows, a lot of people suggested this way as an answer!